ansible/roles/umount-volume/tasks/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+- set_fact: ansible_python_interpreter=/usr/bin/python
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version < '5')
+
+- name: if exists, remove additional mounts
+  mount: name="{{ item.name }}" src="{{ item.source }}" fstype="{{ item.fstype }}" state=unmounted
```

Comment:
The item keyword is reserved for when you pass in a list of items to iterate on. You need the `with_tems` key word and a list data object to be passed in. See Matt's example for [reference].(https://github.com/iPlantCollaborativeOpenSource/atmosphere-ansible/blob/master/ansible/roles/atmo-setup-mount-point/tasks/main.yml#L19-L23)

OR! if you know that you are only unmounting one point, you need to define `name={{ MOUNTNAME }} src={{ MOUNTSOURCE }} fstype={{ MOUNTTYPE }} state=mounted`

and just change the code to 
```
mount: name="{{ MOUNTNAME}}" src="{{ MOUNTSOURCE }}" fstype="{{ MOUNTTYPE}}" state=unmounted
when: (ansible_devices.{{ MOUNTSOURCE }} is defined) and (already_mounted is not defined)
```